rmgpy/rmg/input.py

```diff
@@ -306,7 +365,13 @@ def simple_reactor(temperature,
     termination = []
     if terminationConversion is not None:
         for spec, conv in terminationConversion.items():
-            termination.append(TerminationConversion(species_dict[spec], conv))
+            # check whether key is in species_dict (not if molecule cut provided)
+            if spec not in species_dict:
+                # select first fragment as species used for terminationConversion
+                repr_frag = sorted(mol_to_frag[spec].keys())[0]
+                termination.append(TerminationConversion(species_dict[repr_frag], conv))
+            else:
+                termination.append(TerminationConversion(species_dict[spec], conv))
```

Comment:
Based on the current cutting function, the fragment which has functional group (specific pattern) will be returned in the front order so that the first fragment will be more reactive. However, I do agree that it is better to develop another function to determine which fragment is more suitable to be the representative one.